irteus/pqp.l

```diff
@@ -35,22 +35,24 @@
    (let ((m (pqpmakemodel))
 	 vs v1 v2 v3 (id 0))
      (setf (get self :pqpmodel) m)
```

Comment:
You can return-from this function in case of fs is nil.
```
      (when (null fs)
        (return-from :make-pqpmodel m))
```

Reply:
Thank you for your review.
I force-pushed the commit.